roles/webserver/templates/unicorn.service.j2

```diff
@@ -7,7 +7,7 @@ Description=Open Food Network unicorn server
 Type=forking
 User=openfoodnetwork
 WorkingDirectory={{ current_path }}
-Environment=RAILS_ENV={{ rails_env }}
+EnvironmentFile={{ shared_path }}/unicorn_environment
```

Comment:
:ok_hand:

Reply:
As a side note here, we define some ENV vars for the openfoodnetwork user in `/etc/default/openfoodnetwork`, but as far as I can tell they don't get refreshed when they're re-provisioned. We should re-assess that soon and stop doing it / improve it. It's a separate issue though.

Reply:
I'd add a comment to this line briefly explaining why RAILS_ENV doesnt work.